src/api/firebase.js

```diff
@@ -133,3 +114,74 @@ export async function matchToken(listId) {
 	const jointListTokenSnapshot = await getDocs(jointListTokenQuery);
 	return jointListTokenSnapshot;
 }
+
+export function comparePurchaseUrgency(items) {
+	const overdueList = [];
+	const soonList = [];
+	const kindOfSoonList = [];
+	const notSoonList = [];
+	const purchasedList = [];
+	const inactiveList = [];
+
+	items.forEach((item) => {
+		const differenceTillNextPurchase = item.dateLastPurchased
+			? getDaysBetweenDates(item.dateLastPurchased, item.dateNextPurchased)
+			: getDaysBetweenDates(item.dateCreated, item.dateNextPurchased);
+
+		item.days = differenceTillNextPurchase;
+
+		if (differenceTillNextPurchase >= 60) {
+			item.urgency = 'Inactive';
+		} else if (differenceTillNextPurchase < 0) {
```

Comment:
<img width="465" alt="Screen Shot 2022-11-23 at 9 22 23 PM" src="https://user-images.githubusercontent.com/44907238/203700709-3d9849af-7779-48b9-9841-41676d326749.png">

Reply:
Is there a case when you reach this condition `(differenceTillNextPurchase >= 60`?